apps/web/src/services/asset-inventory/components/CollectorCreateStep4.vue

```diff
@@ -153,9 +155,13 @@ const handleConfirmCreateCollector = async () => {
     try {
         state.collectLoading = true;
         // After the collector created, if the user clicks the collect button, the collector will be executed.
-        await SpaceConnector.client.inventory.collector.collect({
-            collector_id: state.createdCollectorId,
-        });
+        if (state.createdCollectorId) {
+            await SpaceConnector.clientV2.inventory.collector.collect<CollectorCollectParameters, JobModel>({
+                collector_id: state.createdCollectorId,
+            });
+        } else {
+            throw new Error('createdCollectorId is undefined');
```

Comment:
Suggestion:
How about making the modification like this?
 `console.error(new Error('createdCollectorId is undefined'));`